sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/TransformWithStateExec.scala

```diff
@@ -92,6 +95,24 @@ case class TransformWithStateExec(
     }
   }
 
+  private def getDriverProcessorHandle: DriverStatefulProcessorHandleImpl = {
```

Comment:
Lets add a comment to explain where this is run and what we are trying to collect ?